windows/src/engine/kmcomapi/processes/keyboard/Keyman.System.Process.KPInstallKeyboardLanguage.pas

```diff
@@ -236,6 +236,23 @@ function TKPInstallKeyboardLanguage.ConvertBCP47TagToLangID(Locale: string;
 
   LangID := LocaleNameToLCID(PWideChar(Locale), 0);
 
+  if (PRIMARYLANGID(LangID) = 0) and (
+    (SUBLANGID(LangID) < 8) or
+    (SUBLANGID(LangID) > 11)
+  ) then
+  begin
+    // LocaleNameToLCID can return a language code that is 'transient' but
+    // outside of the supported range of $2000, $2400, $2800, $2C00. For
+    // example, we have seen values of $3000, $3400, $3800.
+    // Set-WinUserLanguageList refuses to play ball with these values, so we
+    // reject them as invalid.
+    //
+    // This may be a bug in Windows? TODO: Investigate further and report to
+    // Microsoft
```

Comment:
Worth logging in sentry?

Reply:
I think it probably doesn't need logging at this point. But not in Sentry anyway, surely?!?